src/wallet/scriptpubkeyman.cpp

```diff
@@ -1088,15 +1088,16 @@ void LegacyScriptPubKeyMan::AddKeypoolPubkeyWithDB(const CPubKey& pubkey, const
     m_pool_key_to_index[pubkey.GetID()] = index;
 }
 
-void LegacyScriptPubKeyMan::KeepDestination(int64_t index)
+void LegacyScriptPubKeyMan::KeepDestination(int64_t nIndex)
```

Comment:
Nit: in the previous commit you renamed `nIndex` to `index`.

Reply:
Fixed